src/SFA.DAS.AdminService.Web/Handlers/Gateway/GetApplicationOverviewHandler.cs

```diff
@@ -162,7 +162,7 @@ public async Task<RoatpGatewayApplicationViewModel> Handle(GetApplicationOvervie
             if (application.GatewayReviewStatus.Equals(GatewayReviewStatus.New))
             {
                 // NotRequired checks
-                var TradingNameAndWebsitePage = await _qnaApiClient.GetPageBySectionNo(request.ApplicationId, 0, 1, RoatpQnaConstants.RoatpSections.Preamble.SectionId.ToString());
+                var TradingNameAndWebsitePage = await _qnaApiClient.GetPageBySectionNo(request.ApplicationId, 0, 1, RoatpQnaConstants.RoatpSections.Preamble.PageId);
```

Comment:
Replace magic numbers with constants

Reply:
We have discussed this with Mark. I don’t thing that there is real need for replacing sequenceNo/sectionNo, which clearly can be seen from the signature of the method with the very long wording RoatpQnaConstants.RoatpSections.SectionOne.

Reply:
Because it's obvious to the reader that 0 means Preamble without looking it up in a spreadsheet?